File: programmieren lernen/python/quelldateien/03_wuerfel.py
# Ein Programm, das so oft eine Zahl würfelt, bis der Spieler "stopp" eintippt

import random

while True:
    zahl = random.randint(1, 6)
    zahl = str(zahl)
    
    print("Ich habe eine " + zahl + " gewürfelt.")

    befehl = input()
    
    if befehl == "stopp":
        print("Schade, schon vorbei.")
        break

File: programmieren lernen/cpp/quelldateien/03_wuerfel.cpp
#include <iostream>
#include <stdlib.h>

using namespace std;

int main(void){ 

	srand(time(NULL));
	int zufall;
	string schleife;

	do{
		
		zufall = rand() % 6 + 1;

		cout << endl << "Ich habe eine " << zufall << " gewuerfelt!" << endl;
		
		cin >> schleife;

	}while(schleife != "stopp");

	cout << endl << "Schade, schon vorbei." << endl;

        return 0;
}

